api/handler/put.go

```diff
@@ -519,8 +519,10 @@ func (h *handler) getNewEAclTable(r *http.Request, bktInfo *data.BucketInfo, obj
 	}
 
 	parentAst := tableToAst(bacl.EACL, objInfo.Bucket)
+	strCID := bacl.Info.CID.EncodeToString()
+
 	for _, resource := range parentAst.Resources {
-		if resource.Bucket == bacl.Info.CID.String() {
+		if resource.Bucket == strCID {
```

Comment:
Can we break on true? @alexvanin @KirillovDenis

Reply:
No. The same as previous